service/src/test/java/uk/nhs/adaptors/gp2gp/gpc/GpcDocumentTranslatorTest.java

```diff
@@ -0,0 +1,73 @@
+package uk.nhs.adaptors.gp2gp.gpc;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.when;
+
+import java.io.IOException;
+import java.time.Instant;
+
+import uk.nhs.adaptors.gp2gp.common.service.FhirParseService;
+import uk.nhs.adaptors.gp2gp.common.service.RandomIdGeneratorService;
+import uk.nhs.adaptors.gp2gp.common.service.TimestampService;
+import uk.nhs.adaptors.gp2gp.ehr.EhrDocumentMapper;
+import uk.nhs.adaptors.gp2gp.utils.ResourceTestFileUtils;
+
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.mockito.junit.jupiter.MockitoSettings;
+import org.mockito.quality.Strictness;
+
+@ExtendWith(MockitoExtension.class)
+@MockitoSettings(strictness = Strictness.LENIENT)
+public class GpcDocumentTranslatorTest {
+    private static final String TEST_FILE_DIRECTORY = "/ehr/request/fhir/";
+    private static final String INPUT_DIRECTORY = "input/";
+    private static final String OUTPUT_DIRECTORY = "output/";
+    private static final String INPUT_PATH = TEST_FILE_DIRECTORY + INPUT_DIRECTORY;
+    private static final String OUTPUT_PATH = TEST_FILE_DIRECTORY + OUTPUT_DIRECTORY;
+    private static final String BINARY_INPUT_FILE = "test-binary.json";
+    private static final String EXPECTED_MHS_OUTBOUND_REQUEST_FILE = "expected-mhs-outbound-request-payload.json";
+    private static final String TEST_DOCUMENT_ID = "test-document-id";
+    private static final String MESSAGE_ID = "message-id";
+    private static final String TEST_ID = "test-id";
+    private static final String TEST_DATE_TIME = "2020-01-01T01:01:01.01Z";
```

Comment:
This is not a great value to use when testing. What if the date format had the position of hours and minutes swapped. Would this test failed? Or the DD vs dd issue we saw earlier.

Reply:
It is only for testing purposes so we would have a constant value to assert.